src/content/components/ActiveRSMessages/ActiveRSMessages.tsx

```diff
@@ -0,0 +1,257 @@
+import * as React from "react";
+import { getTargetingAttributes } from "./TargetingParser";
+import styles from "./ActiveRSMessages.scss";
+import { columnTransforms } from "../BugList/columnTransforms";
+import {
+  fetchRemoteSettingsMessages,
+  fetchBugById,
+} from "../../../server/queryUtils";
```

Comment:
I didn't really intend for `queryUtils` to be imported by content files... but it does appear to work 🤷‍♀ Obviously the API key won't be available and the use of `request` is kind of unnecessary, other than that I guess there's not much of an issue